### DISCUSSION TITLE
OAuth/ Keycloak access

### DISCUSSION THREAD

**Discussion by Syndlex** [strimzi/strimzi-canary#222] [Q&A]
How do i use canary on a listener with the keycloak plugin enabled?

I have the feeling this is not Implemented? 

I have not fully understand the SASL configuration.

**Comment by scholzj** [comment]
@ppatierno any idea?

**Reply by ppatierno** [reply]
The SASL mechanisms supported by canary are PLAIN, SCRAM-SHA 256 and 512. There is no support for something like OAuth.

**Comment by Neustradamus** [comment]
It is possible to rename the title?
Oatuh -> OAuth
Thanks in advance.
